pom.xml

```diff
@@ -1152,7 +1152,6 @@
                     <version>${cs.surefire-plugin.version}</version>
                     <configuration>
                         <useSystemClassLoader>false</useSystemClassLoader>
-                        <argLine>-Djava.security.egd=file:/dev/./urandom -noverify</argLine>
```

Comment:
Turns out this argline override, essentially disables any quality plugins that try to export their config to the mvn/cmd line. Disabling this now that linux and our build/pkgs are much faster. In my testing, I found no difference in the amount of time it took for the build.